src/functions/template-tags/html.php

```diff
@@ -127,8 +127,79 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string The rel attribute.
+ */
+function tribe_rel( $rel, $echo = true ) {
+	if ( empty( $rel ) ) {
+		return;
+	}
+
+	if ( is_array( $rel )) {
+		$rel = implode( ' ', $rel );
+	}
+
+	$attr = 'rel="' . esc_attr( $rel ) . '"';
+
+	if ( ! $echo ) {
+		return $attr;
+	}
+
+	echo $attr;
+}
+
+/**
+ * Adds target attribute for links.
+ * If target is '_blank' also adds appropriate rel attributes.
+ *
+ * @since TBD
+ *
+ * @param string       $target Value for the target attribute.
+ * @param array|string $rel    Value(s) for rel attribute.
+ * @param boolean      $echo   Whether to echo the string or return it.
+ *
+ * @return string The target attribute. Potentially also the rel attribute.
+ */
+function tribe_target( $target, $rel = [], $echo = true ) {
+	if ( empty( $target ) ) {
+		return;
+	}
+
+	// Format $rel as an array.
+	if ( ! is_array( $rel ) ) {
+		$rel = explode( ' ', $rel );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 83f83ec726979e5dac1ae65161f94ab5</sub>